core/src/main/java/de/jplag/reporting/reportobject/model/OverviewReport.java

```diff
@@ -8,6 +8,8 @@
 
 public record OverviewReport(
 
+        @JsonProperty("version") Version version,
```

Comment:
Maybe we name this property rather `JPlag-version` or similar? I could image in the future to introduce an actual versioning of the JSON report structure, and then we would have two version numbers: version of the json format and the version of JPlag used to generate it.